projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -161,9 +170,9 @@
   "item.pamhc2foodextended.babaganoushitem": "烤茄泥酱",
   "item.pamhc2foodextended.baconpancakesitem": "培根薄煎饼",
   "item.pamhc2foodextended.baconwrappedchiliitem": "辣椒培根卷",
-  "item.pamhc2foodextended.baconwrappeddatesitem": "枣椰培根卷",
+  "item.pamhc2foodextended.baconwrappeddatesitem": "椰枣培根卷",
   "item.pamhc2foodextended.bakedalaskaitem": "火焰冰淇淋蛋糕",

```

```suggestion
  "item.pamhc2foodextended.bakedalaskaitem": "热烤阿拉斯加",
```
据维基百科：热烤阿拉斯加（Baked Alaska）也译作焗火焰雪山、阿拉斯加雪球或烤冰淇淋蛋白甜饼，是一道冰淇淋点心。

---

这个算是为避免名称难以理解，不过改也行